src/modules/menu/components/activeLayers/MenuActiveLayersListItem.vue

```diff
@@ -64,7 +64,7 @@ const id = computed(() => layer.value.id)
 const attributionName = computed(() =>
     layer.value.attributions.map((attribution) => attribution.name).join(', ')
 )
-const showLegendIcon = computed(() => layer.value.hasLegend)
+const showLegendIcon = computed(() => layer.value.hasDescription)
```

Comment:
We could maybe go all the way and rename this to `showLayerDescriptionIcon` and also rename the `LayerLegendPopup` to `LayerDescriptionPopup` so that we are thorough with our renaming